android/src/main/java/com/jhomlala/better_player/BetterPlayer.java

```diff
@@ -122,6 +125,71 @@
         setupVideoPlayer(eventChannel, textureEntry, result);
     }
 
+    static void preCache(Context context, String key, String dataSource, long preCacheSize, long maxCacheSize, long maxCacheFileSize, Map<String, String> headers, Result result) {
```

Comment:
Why it's static? Can you format this line so it will break after 80 chars?

Reply:
It's static because an instance of BetterPlayerController is not required. This is more convenient that way, because it allows clean pre-caching anywhere in the app (I call the method from a ViewModel), without having to think about disposing the object. Generally speaking, methods should be static if they can be.